configure_user/main.tf

```diff
@@ -0,0 +1,29 @@
+data "ibm_iam_auth_token" "tokendata" {}
+
+locals {
+  sensitive_tokendata = sensitive(data.ibm_iam_auth_token.tokendata.iam_access_token)
+}
+
+resource "null_resource" "configure_user" {
+
+  triggers = {
+    always_run = timestamp()
+  }
+
+  provisioner "local-exec" {
```

Comment:
We'll need one to do the reverse action on destroy - no necessary in this PR but in future

Reply:
This is something to discuss with watson team. I don't have a strong opinion on this but IMHO since the user can have multiple ML instances, if I delete the profile, I may cause issues to other projects\ML instances.

I agree to do in another PR in case